apps/mobile/index.js

```diff
@@ -46,6 +46,7 @@ LogBox.ignoreLogs([
   'Overriding previous layout animation',
   'There was a problem sending log message',
   'Constants.manifest has been deprecated in favor of Constants.expoConfig',
+  "CommentsBottomSheet_comments_connection' doesn't exist.",
```

Comment:
Ignore the connection warning. Seems like the reason the comment didn't show up because of missing connection. 

At some cases, the connection didn't initialise yet and we get the warning. The reason is the bottom sheet modal is using `lazyLoadQuery` so the connection will init when the bottom sheet is open. 

The warning only appear when we submit the comment without open the bottom sheet

Reply:
gotcha! this wouldn't happen in a normal use case then right? would you be able to add a comment to the file so we can keep the context?

maybe something like 
`Warning only occurs when commenting before opening the bottom sheet because it uses `lazyLoadQuery`, which initializes the connection` ?

Reply:
@kaitoo1 Yup! Normal use case's working fine. 

Adding the comment above the log

Reply:
nice! tyyy